modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/utils.py

```diff
@@ -565,6 +566,42 @@ def mapper(at):
     return df
 
 
+def arrow_type_to_pandas(at: pa.lib.DataType):
+    """
+    Convert the specified arrow type to pandas dtype.
+
+    Parameters
+    ----------
+    at : pa.lib.DataType
+
+    Returns
+    -------
+    dtype
+    """
+    if at == pa.string():
+        return _get_dtype(str)
+    return at.to_pandas_dtype()
+
+
+def get_cat_value_dtype(cdt: Union[LazyProxyCategoricalDtype, pandas.CategoricalDtype]):
```

Comment:
maybe it's better to move this function to a non-hdk specific module? [`modin/core/dataframe/pandas/metadata/dtypes.py`](https://github.com/modin-project/modin/blob/master/modin/core/dataframe/pandas/metadata/dtypes.py)